### ISSUE TITLE
BUG - Joining a call with background-blur enabled shows a black screen camera on v0.40 and above

### ISSUE THREAD

**Issue by carloscheddar** [daily-co/daily-js#215]
# Expected behavior
Joining the call should keep the camera showing with the background-blur as seen on the Lobby.

# Describe the bug (unexpected behavior)
When calling `.join()` with background-blur enabled the camera shows a black screen instead of video.

# Steps to reproduce
1. Enable `background-blur` after initializing the call object
2. Join the call with blur enabled
3. See a black screen instead of yourself with no errors on the console

# Screenshots

![Screenshot 2023-03-23 at 4 32 02 PM](https://user-images.githubusercontent.com/3360539/227347674-ae3ac770-b804-47fb-973a-7b0d7c228985.png)

# System information 

 

* Device: Macbook Pro 14"
* OS, version: Mac OS, 13.2
* Browser, version: Chrome, latest

# Additional context
Versions of Daily.js up to `0.38` work as expected. Using the new inputSettings via callObject or join properties does not fix the issue.

**Comment by mattieruth** [comment]
We have not been able to reproduce the issue. We tried the following:

```
let call = DailyIframe.createCallObject({
    dailyConfig: {
        experimentalChromeVideoMuteLightOff: true,
    },
});
call.startCamera();
call.updateInputSettings({ video: {processor: {type: 'background-blur'}});
call.join({url: <>});
```
on both 0.40. and 0.41 both with calling `startCamera()` and without calling it.  On 0.40, we see a bug where the background blur does not apply. This is resolved in 0.41.0. But we have not been able to reproduce a black video. Is there another setting or call we should try?

**Comment by Agular** [comment]
Hi, we had a case with the blackscreen too, but not due to background blur, but probably because of ISP network configuration. We had to change sfu_switchover to 0.5 to avoid P2P connections which solved the black screen for us.
https://docs.daily.co/reference/rest-api/rooms/create-room#sfu_switchover
Cheers.
